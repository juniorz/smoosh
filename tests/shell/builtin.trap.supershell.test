trap 'echo bye' EXIT
(trap)
(trap 'echo so long' EXIT; trap)
(trap)
