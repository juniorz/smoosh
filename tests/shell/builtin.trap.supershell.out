trap -- 'echo bye' EXIT
trap -- 'echo so long' EXIT
so long
trap -- 'echo bye' EXIT
bye
